tests/test_conv_int.py

```diff
@@ -0,0 +1,811 @@
+        (15, 'Reactor1', 4, 'core',  922350000,
+         0.04, 751553292.748, 2.19249259917e+28),
+        (15, 'Reactor1', 4, 'spent', 922350000,
+         0.0265486725664, 498818557.134, 1.45519420299e+28),
+        (15, 'Reactor1', 4, 'spent', 942390000,
+         0.0530973451327, 2.91305071939e+13, 9.53273565408e+32)
+    ], dtype=ensure_dt_bytes([
+        ('AgentId', '<i8'), ('Prototype', 'O'), ('Time', '<i8'),
+        ('InventoryName', 'O'), ('NucId', '<i8'), ('Quantity', '<f8'),
+        ('Activity', '<f8'), ('DecayHeat', '<f8')
+    ]))
+    )
+    assert_frame_equal(cal, refs)
+
+
+@dbtest
+def test_convint_get_inventory_timeserie(db, fname, backend):
+    myEval = cym.Evaluator(db)
+    cal = com.get_inventory_timeserie(myEval)
+    exp_head = ['Time', 'Quantity']
+    assert_equal(list(cal), exp_head)  # CHeck we have the correct headers
+
+    cal = com.get_inventory_timeserie(myEval, fac_list=['Reactor1'],
+                               nuc_list=['94239'])
+    refs = pd.DataFrame(np.array([
+        (0, 0.0            ),
+        (1, 0.0444814879803),
+        (2, 0.0621806030246),
+        (3, 0.0798797180688),
+        (4, 0.0530973451327)
+    ], dtype=ensure_dt_bytes([
+        ('Time', '<i8'), ('Quantity', '<f8')
+    ]))
+    )
+    assert_frame_equal(cal, refs)
+
+    cal=com.get_inventory_timeserie(myEval, fac_list=['Reactor1'],
+                               nuc_list=['94239', '92235'])
+    refs=pd.DataFrame(np.array([
+        (0, 0.0            ),
+        (1, 0.0460607124057),
+        (2, 0.0726093849721),
+        (3, 0.0991580575384),
+        (4, 0.119646017699 )
+    ], dtype=ensure_dt_bytes([
+        ('Time', '<i8'), ('Quantity', '<f8')
+    ]))
+    )
+    assert_frame_equal(cal, refs)
+
+
+@dbtest
+def test_convint_get_inventory_activity_timeserie(db, fname, backend):
+    myEval=cym.Evaluator(db)
+    cal=com.get_inventory_activity_timeserie(myEval)
+    exp_head=['Time', 'Activity']
+    assert_equal(list(cal), exp_head)  # CHeck we have the correct headers
+
+    cal=com.get_inventory_activity_timeserie(myEval, fac_list=['Reactor1'],
+                               nuc_list=['94239'])
+    refs=pd.DataFrame(np.array([
+        (0, 0.0             ),
+        (1, 2.44036364223e+13),
+        (2, 3.41138054869e+13),
+        (3, 4.38239745515e+13),
+        (4, 2.91305071939e+13)
+    ], dtype=ensure_dt_bytes([
+        ('Time', '<i8'), ('Activity', '<f8')
+    ]))
+    )
+    assert_frame_equal(cal, refs)
+
+    cal=com.get_inventory_activity_timeserie(myEval, fac_list=['Reactor1'],
+                               nuc_list=['94239', '92235'])
+    refs=pd.DataFrame(np.array([
+        (0, 0.0             ),
+        (1, 2.4403666094e+13),
+        (2, 3.41140014315e+13),
+        (3, 4.3824336769e+13),
+        (4, 2.91317575657e+13),
+    ], dtype=ensure_dt_bytes([
+        ('Time', '<i8'), ('Activity', '<f8')
+    ]))
+    )
+    assert_frame_equal(cal, refs)
+
+
+@dbtest
```

Comment:
I'm curious about the purpose of the function decorators here. Maybe you can elaborate on them during the code review meeting?